_std/macros/atmos.dm

```diff
@@ -221,13 +221,13 @@ What can break when adding new gases:
 	TEG stats computer will ignore your new gas. Feel free to add it to reactor_stats.dm manually but good luck.
 */
 
-#define SPECIFIC_HEAT_PLASMA	200
-#define SPECIFIC_HEAT_O2		20
-#define SPECIFIC_HEAT_N2		20
-#define SPECIFIC_HEAT_CO2		30
+#define SPECIFIC_HEAT_PLASMA	150
+#define SPECIFIC_HEAT_O2		40
+#define SPECIFIC_HEAT_N2		50
+#define SPECIFIC_HEAT_CO2		40
 #define SPECIFIC_HEAT_FARTS 	69
-#define SPECIFIC_HEAT_RADGAS 	20
-#define SPECIFIC_HEAT_N2O		40
+#define SPECIFIC_HEAT_RADGAS 	5
+#define SPECIFIC_HEAT_N2O		60
```

Comment:
What was the basis for the new numbers? I think I've seen some take since showing similar specific heart values for some of the gases.

Reply:
the basis was the isochoric heat capacity of the various gasses doubled and rounded to the nearest multiple of 10, with a bit of fudging to add some variety. It was not very scientific.

I've been kinda thinking maybe it would've been better to just nerf plasma all the way down to 100, instead of doubling the others, but that's a pretty huge nerf.